examples/requirements-examples.txt

```diff
@@ -2,3 +2,4 @@ torch==1.8.1
 tensorflow
 keras-tuner
 torchvision
+sklearn
```

Comment:
How is this related to the issue you’re solving?

Reply:
@SaboniAmine Do you think it's a problem to add an Sklearn test ?

Sklearn is smaller that Pytorch so it may be useful.